src/main/java/com/smartcar/sdk/Vehicle.java

```diff
@@ -162,9 +165,8 @@ public SmartcarResponse location() throws SmartcarException {
    * @throws SmartcarException if the request is unsuccessful
    */
   public void unlock() throws SmartcarException {
-    RequestBody body = new FormBody.Builder()
-      .add("action", "UNLOCK")
-      .build();
+    String json = "{\"action\": \"UNLOCK\"}";
```

Comment:
is there a better way to build this without using a string? I think building a `JSONObject` would be better:

```java
JSONObject json = Json.createObjectBuilder()
  .add("action", "UNLOCK")
  .build();
RequestBody body = RequestBody.create(JSON, json.toString());
```

I think that would work, probably have to check if the `.toString()` just works the way we'd want it to.

Reply:
Wan't aware this existed. I'll change it for lock and unlock.